tools/biddersettings.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Brightcove Prebid Plugin : Bidder Settings JSON Converter</title>
+    <style type="text/css">
+        <!--
```

Comment:
I think we don't need this comment here.

Reply:
Good catch, fixed.